autogen/convert_structs.py

```
@@ -228,6 +233,7 @@ def parse_struct(struct_str, sortFields = False):
 
         field = {}
         field['type'] = field_type.strip()
+        if field_enum: field['enum'] = field_enum
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)